Cargo.toml

```diff
@@ -46,7 +46,7 @@ test-cuda = ["cuda"]
 [dev-dependencies]
 rand = "0.8.5"
 tempfile = "3.3.0"
-mnist = "0.5.0"
+mnist = {version = "0.5.0", features = ["download"]}
```

Comment:
Also yeah - the github workflow errors were why I didn't add this feature to begin with, since it doesn't work on all OS. Likely will be adding mnist & other datasets directly to dfdx in the future with #156 so we don't need to worry about that.

In the mean time, if you want to contribute the fix to the mnist crate, that'd be great, but it looks like it hasn't been updated in a while on crates.io

Reply:
I think that this add should actually work on windows, and that the github action fails because it tries to build for the wrong target. If you look at the [file the compile error occurs in] (https://github.com/davidMcneil/mnist/blob/master/src/download.rs) it should have support for windows.

Reply:
I think that may be due to the fix not being re-released on crates.io? The last comment on the open issue about it asks about a new release. It seems like the 0.5.0 release was nov 8th, 2021, and the fix was merged in 2022

Reply:
Yes, I think that's the case. I just sent a message to the maintainer to see if they might be willing to publish that patch; it might take a bit for that to come through, however, so I'll keep an eye on it and circle back when there's an update

Reply:
Sounds good, thanks for reaching out